tasks/scheduler.go

```diff
@@ -442,7 +441,8 @@ func (s *scheduler) prepareTask(task *deliverTxTask) {
 }
 
 // executeTask executes a single task
-func (s *scheduler) executeTask(task *deliverTxTask) {
+func (s *scheduler) executeTask3(task *deliverTxTask) {
```

Comment:
is this just for preserving the old impl?

Reply:
nope- this was an oversight that needs removing :)